webtool/test_transcribe.py

```
@@ -1627,6 +1646,27 @@ def _ki_projekt(monkeypatch, tmp_path, name, bases=("S1", "S2")):
 
 
+def test_ki_projekt_kann_diarize_marken_drucken(monkeypatch, tmp_path, capsys):
+    """#443-Waechter: die Fixture KANN die echten Marken — die Faehigkeit darf nicht
+    still verschwinden. Genau eine durchgaengig stumme Attrappe nahm jedem Test den
+    Sensor fuer die [active]/[done]-Buchfuehrung; der erste #418-Fix lief daran
+    unbemerkt wirkungslos vorbei (14 Tests, alle gruen)."""
+    _ki_projekt(monkeypatch, tmp_path, "FixturDemo", diarize_druckt=True)
+    from webtool import correct
+    correct.cmd_diarize("FixturDemo", ["S1", "S2"])
+    out = capsys.readouterr().out
+    assert "[active] S1" in out and "[done] S2" in out
```

Ruff (0.16.2)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)